config/var_production.env

```diff
@@ -0,0 +1,12 @@
+DOCS_BASE_PATH=/api
+DOCS_EXTERNAL_DESCRIPTION="Documentation for using the Platform APIs"
+DOCS_EXTERNAL_URL=https://docs.google.com/document/d/1p3q9OT9latXqON20WDh4CNPxIShUunfGgqT163r-Caw/edit?usp=sharing
+DOCS_HOST=qa-platform.nypl.org
+DOCS_INFO_DESCRIPTION="Endpoints require an OpenID Connect token from the NYPL OAuth server and interact with NYPL Data Streams."
+DOCS_INFO_TITLE="NYPL Platform APIs"
```

Comment:
Should we move common config to `config/var_app`? Or at least just those config params we're very unlikely to change independently?
 * `DOCS_INFO_DESCRIPTION`
 * `DOCS_INFO_TITLE`

Reply:
Ok I've moved these two, there are potentially others that could be moved but I'd like to proceed carefully here since I don't necessarily know which params we might want to change